### PR TITLE
feat(schedule_service): implement UpdateBackupSchedule

### PR DESCRIPTION
**Pull request by ulya-sidorina**

### CODE REVIEW

File: internal/connectors/db/yql/queries/write.go [hunk 1]
```diff
@@ -335,16 +335,11 @@ func BuildUpdateBackupScheduleQuery(schedule types.BackupSchedule, index int) Wr
...
-	if len(schedule.SourcePaths) > 0 {
-		d.AddValueParam("$paths", table_types.StringValueFromString(strings.Join(schedule.SourcePaths, ",")))
-	}
-	if len(schedule.SourcePathsToExclude) > 0 {
-		d.AddValueParam(
-			"$paths_to_exclude",
-			table_types.StringValueFromString(strings.Join(schedule.SourcePathsToExclude, ",")),
-		)
-	}
+	d.AddValueParam("$paths", table_types.StringValueFromString(strings.Join(schedule.SourcePaths, ",")))
```
Comment (qrort) [inline internal/connectors/db/yql/queries/write.go:338]: We need to somehow verify that for each `WithUpdateBackupSchedule` call, correct `SourcePaths` and  `SourcePathsToExclude` are set. It is hard, maybe we need some kind of integration test for schedules, that verifies that non-empty `SourcePaths` are not overrided by empty strings

Comment (qrort) [reply internal/connectors/db/yql/queries/write.go:338]: I looked through the code and it seems that it is ok for now, but we need a ticket for adding schedules to integration test

**Review by qrort — APPROVED** [review]
